### PR TITLE
Sync latest CRDs to helm chart

### PR DESCRIPTION
**Pull request by e0ne**

### CODE REVIEW

**Review by moshe010 — APPROVED** [review]

File: deployment/sriov-network-operator/templates/leader_election_role.yaml [hunk 1]
```diff
@@ -0,0 +1,41 @@
+# permissions to do leader election.
```
Comment (adrianchiris) [inline deployment/sriov-network-operator/templates/leader_election_role.yaml:1]: i dont think this is used today when running `make deploy-k8s`

IMO you need to update the role.yaml template

https://github.com/k8snetworkplumbingwg/sriov-network-operator/blob/348e699badfaf15d30fe2c68bfd1f07a4b0e0c10/deploy/role.yaml#L86

perhaps other `.yaml` files from [deploy ](https://github.com/k8snetworkplumbingwg/sriov-network-operator/tree/master/deploy) folder are out of sync

Comment (e0ne) [reply deployment/sriov-network-operator/templates/leader_election_role.yaml:1]: Good catch, I'll update my PR with it

**Review by adrianchiris — APPROVED** [review]
/lgtm

**Comment by adrianchiris** [comment]
@pliurh @zshi-redhat can this be merged ?

**Pull request merged**